plutus-streaming/src/Cardano/Api/Extras.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Cardano.Api.Extras where
+
+import Cardano.Api (BlockHeader (BlockHeader), BlockNo, ChainPoint (ChainPoint, ChainPointAtGenesis),
+                    HasTypeProxy (proxyToAsType), Hash, SerialiseAsRawBytes (deserialiseFromRawBytes), ToJSON)
+import Data.ByteString.Base16 qualified as Base16
+import Data.ByteString.Char8 qualified as C8
+import Data.Proxy (Proxy (Proxy))
+import Data.String (IsString (fromString))
+import GHC.Generics (Generic)
+
+-- FIXME orphan instance
+-- https://github.com/input-output-hk/cardano-node/pull/3608
+instance IsString (Hash BlockHeader) where
```

Comment:
This instance is in `Cardano.Api` now https://github.com/input-output-hk/cardano-node/pull/3619